frontend/src/scenes/project-homepage/ProjectHomepage.scss

```diff
@@ -17,6 +17,16 @@
                     margin: 0;
                 }
             }
+
+            .posthog-3000 & {
+                a {
```

Comment:
This feels very generic a CSS change to me. What were you hoping to achieve with it? Probably worth adding a specific class to the element you're trying to target

Reply:
It's scoped to this:

<img width="533" alt="image" src="https://github.com/PostHog/posthog/assets/154479/de753160-d4d8-4d0a-be8f-c314cd36e2a0">

Makes this the text color instead of link color:

<img width="434" alt="image" src="https://github.com/PostHog/posthog/assets/154479/b58939b4-eec0-4d10-a688-38a3df3ef4a8">